README.md

```diff
@@ -130,14 +152,24 @@ to handle updating ads on progressive web applications:
 
 Each time a new advertisement is requested, the AdSense parameter `data-ad-region` is
 updated to a random value. For this reason, you cannot set the `data-ad-region` attribute
-on the `<adsbygoogle />` component.
-
-For test mode, the following blog was used as a reference:
```

Comment:
This is still applied in the component, right?

Reply:
Yes it is still applied. I just Capitalized the component name and removed the link to the blog that 404s here.